src/main/java/uk/gov/hmcts/reform/em/EmTestConfig.java

```diff
@@ -18,12 +14,7 @@
 
     @Bean
     RestTemplate restTemplate() {
-        SimpleClientHttpRequestFactory requestFactory = new SimpleClientHttpRequestFactory();
-
-        Proxy proxy = new Proxy(Proxy.Type.HTTP, new InetSocketAddress("proxyout.reform.hmcts.net", 8080));
-        requestFactory.setProxy(proxy);
-
-        return new RestTemplate(requestFactory);
+        return new RestTemplate();
```

Comment:
Won't spring just give you a an object if there is a constructor with no args? Maybe you don't need to define the Bean anymore